source/_docs/automation/trigger.markdown

```diff
@@ -170,6 +171,16 @@ automation:
     to: "home"
 ```
 
+It's possible to give a list of frome_states or to_states:
+```yaml
+automation:
+  trigger:
+    platform: state
+    entity_id: vacuum.test
+    from: "cleaning", "returning"
```

Comment:
Let's use actual YAML syntax as a good practice in our examples.

```yaml
    from:
      - "cleaning"
      - "returning"
```

or

```yaml
    from: ["cleaning", "returning"]
```

Reply:
Fixed.